it/mail/chrome/messenger/accountCreation.properties

```diff
@@ -1,73 +1,73 @@
 # accountCreation.properties
 
 # LOCALIZATION NOTE(cleartext_warning): %1$S will be the hostname of the server the user was trying to connect to.
-cleartext_warning=%1$S non usa alcun tipo di cifratura.
+cleartext_warning=%1$S non utilizza crittografia.
 # LOCALIZATION NOTE(selfsigned_warning): %1$S will be the hostname of the server the user was trying to connect to.
-selfsigned_warning=%1$S non usa un certificato verificabile.
-selfsigned_details=Generalmente un server sicuro fornisce un certificato in grado di attestarne la reale identità. Il collegamento sarà cifrato ma non sarà possibile verificare l'autenticità del server.
-cleartext_details=Un server di posta non sicuro non utilizza collegamenti cifrati per proteggere password e informazioni personali. Collegandosi a questo server si mette a rischio la sicurezza delle password e dei dati personali.
+selfsigned_warning=%1$S non utilizza un certificato affidabile.
+selfsigned_details=Generalmente un server e-mail sicuro fornisce un certificato affidabile in grado di attestarne la reale identità. La connessione al server e-mail sarà crittografata ma non sarà possibile convalidare l'autenticità del server.
+cleartext_details=I server e-mail non sicuri non utilizzano connessioni crittografate per proteggere le tue password e le tue informazioni personali. Collegandoti a questo server metterai a rischio la sicurezza delle tue password e dei tuoi dati personali.
```

Comment:
CORRECTIONS NEEDED. Too many possessive pronouns are redundant. I would suggest to modify the sentence as follows: 

 +cleartext_details=I server e-mail non sicuri non utilizzano connessioni crittografate per proteggere password e informazioni personali. Collegandoti a questo server metterai a rischio la sicurezza delle tue password e dei tuoi dati personali.